docs/docusaurus.config.js

```diff
@@ -209,77 +209,6 @@ const config = {
           src: "logo.svg",
         },
       },
-      footer: {
-        links: [
-          {
-            title: "Developer",
-            items: [
-              {
-                label: "Docs",
-                to: "/docs/v1",
-              },
-
-              {
-                label: "Templates",
-                href: `https://mastra.ai/templates`,
-              },
-              {
-                label: "Principles of Building AI Agents",
-                href: `https://mastra.ai/book`,
-              },
-
-              {
-                label: "llms.txt",
-                href: `https://mastra.ai/llms.txt`,
-              },
-
-              {
-                label: "llms-full.txt",
-                href: `https://mastra.ai/llms-full.txt`,
-              },
-              {
-                label: "MCP Registry Registry",
-                href: `https://mastra.ai/mcp-registry-registry`,
-              },
-              {
-                label: "Mastra Cloud Status",
-                href: "https://statuspage.incident.io/mastra-cloud",
-              },
-            ],
-          },
-          {
-            title: "Company",
-            items: [
-              {
-                label: "GitHub",
-                href: "https://github.com/mastra-ai/mastra",
-              },
-              {
-                label: "Discord",
-                href: "https://discord.gg/BTYqqHKUrf",
-              },
-              {
-                label: "X",
-                href: "https://x.com/mastra_ai",
-              },
-              {
-                label: "YouTube",
-                href: "https://www.youtube.com/@mastra-ai",
-              },
-            ],
-          },
-          {
-            title: "Legal",
-            items: [
-              {
-                label: "Privacy Policy",
-                href: `https://mastra.ai/privacy-policy`,
-              },
-            ],
-          },
-        ],
-        copyright: `© ${new Date().getFullYear()} Mastra.`,
-      },
```

Comment:
Okay, this might remove the footer from rendering, but I still worry about that `hidden` class in the actual footer template file. Please remove this as well so its a clean change.

--> https://github.com/mastra-ai/mastra/blob/main/docs/src/theme/Footer/Layout/index.tsx#L7